src/view/dataElements/xdmObject/components/sandboxSelector.jsx

```diff
@@ -54,13 +53,11 @@ const SandboxSelector = ({
         }));
       } catch (e) {
         if (e.name !== "AbortError") {
-          reportAsyncError(
-            new UserReportableError(`Failed to load sandboxes.`, {
-              originatingError: e
-            })
-          );
-          return undefined;
+          reportAsyncError(e);
```

Comment:
I moved the creating of UserReportableError up to fetchSandboxes (and did so similarly in other places). It seemed to make the most sense there and reduced duplicate code.